docs/source/developer_guide/contributing_guide.md

```diff
@@ -0,0 +1,102 @@
+# How to Contribute
+
+This documentation provides high-level overview of contributing to cuspatial.
+
+`cuSpatial` is a part of the RAPIDS community.
+Developers should follow the same guidelines that applies to RAPIDS when contributing to cuspatial.
+RAPIDS [contributing section](https://docs.rapids.ai/contributing) walks through the process of identifying an issue,
+submit and merging a PR.
+
+## Directory structure and file naming
```

Comment:
To match the C++ guide, this fits better in something called a Developer Guide, IMO.